pkg/config/msp_test.go

```diff
@@ -1260,14 +1252,15 @@ func TestRevokeCertificateFromMSPFailure(t *testing.T) {
 }
 
 func baseMSP() MSP {
-	certBlock, _ := pem.Decode([]byte(dummyCert))
-	cert, _ := x509.ParseCertificate(certBlock.Bytes)
-
-	privKeyBlock, _ := pem.Decode([]byte(dummyPrivateKey))
-	privKey, _ := x509.ParsePKCS8PrivateKey(privKeyBlock.Bytes)
-
-	crlBlock, _ := pem.Decode([]byte(dummyCRL))
-	crl, _ := x509.ParseCRL(crlBlock.Bytes)
+	cert, privKey := generateCACertAndPrivateKey("org1.example.com")
+	crlBytes, err := cert.CreateCRL(rand.Reader, privKey, nil, time.Now(), time.Now().Add(YEAR))
+	if err != nil {
+		log.Fatalf("Failed to create CRL: %s", err)
```

Comment:
I think @DereckLuo may have a PR related to this but, functions like `baseMSP` are test helpers and can be provided with a `*testing.T`. We should be wiring that through and making assertions as necessary in preference to a panic when appropriate.

This applies to the functions that follow as well.

Reply:
I can make the changes in the same pr.